.cypress/integration/trace_analytics_test/trace_analytics_traces.spec.js

```diff
@@ -366,6 +365,6 @@ describe('Testing traces Custom source', () => {
 
     cy.get('a.euiLink.euiLink--primary').first().click();
     cy.get('[data-test-subj="globalLoadingIndicator"]').should('not.exist');
-    cy.get('.overview-content').should('contain.text', 'be0a3dceda2ecf601fd2e476fef3ee07');
+    cy.get('.overview-content').should('contain.text', 'd5bc99166e521eec173bcb7f9b0d3c43');
```

Comment:
how did this get changed?

Reply:
The date was previously invalid. With the actually date displayed, the default sorting added in changes the order and this test was clicking the first link which is now d5bc99166e521eec173bcb7f9b0d3c43.